sfge/src/main/java/com/salesforce/graph/ops/expander/ApexPathExpanderUtil.java

```diff
@@ -54,8 +51,12 @@ public static List<ApexPath> expand(
             logFilteredOutPath(throwStatementVertex);
             return Collections.emptyList();
         } else {
-            ApexPathExpanderUtil apexPathExpanderUtil = new ApexPathExpanderUtil(config);
-            return apexPathExpanderUtil._expand(g, path, config);
+            final PathExpansionRegistry registry = new PathExpansionRegistry();
+            ApexPathExpanderUtil apexPathExpanderUtil = new ApexPathExpanderUtil(config, registry);
```

Comment:
Probably a dumb question: Why are we instantiating a util? Is it strictly necessary for this Util class to have instance methods?

Reply:
Yeah, "Util" is probably a misnomer since there's a single static entry method. I started with regrouping the class but decided to retract to keep my changes as specific as possible.